### PR TITLE
Weekly maintenance update for the CODEOWNERS.txt file

### PR DESCRIPTION
**Pull request by fbsolo-ms1**
Weekly maintenance update for the

CODEOWNERS.txt

file.
_________________

# Description

# Checklist

- [ ] I have read the contribution guidelines.
  - [General](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
  - [SDK](https://github.com/Azure/azureml-examples/blob/main/sdk/CONTRIBUTING.md)
  - [CLI](https://github.com/Azure/azureml-examples/blob/main/cli/CONTRIBUTING.md)
- [ ] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [ ] Pull request includes test coverage for the included changes.
- [ ] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

**Comment by fbsolo-ms1** [comment]
#sign-off

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -248,15 +242,11 @@
...
 /sdk/python/featurestore_sample/notebooks/sdk_only/1.\ Develop\ a\ feature\ set\ and\ register\ with\ managed\ feature\ store.ipynb @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
 /sdk/python/featurestore_sample/notebooks/sdk_only/2.\ Experiment\ and\ train\ models\ using\ features.ipynb @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
 /sdk/python/featurestore_sample/notebooks/sdk_only/3.\ Enable\ recurrent\ materialization\ and\ run\ batch\ inference.ipynb @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
 /sdk/python/featurestore_sample/notebooks/sdk_only/4.\ Enable\ online\ store\ and\ run\ online\ inference.ipynb @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
-/sdk/python/featurestore_sample/notebooks/sdk_only/5.\ Develop\ a\ feature\ set\ with\ custom\ source.ipynb @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1
-/sdk/python/foundation-models/benchmarking/evaluating_claude_models.ipynb @arun-rajora
```
Comment (sdgilley) [inline .github/CODEOWNERS:257]: Can you move these three lines ABOVE our comment?  We shouldn't delete them.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -281,12 +271,11 @@
 /sdk/python/jobs/single-step/tensorflow/train-hyperparameter-tune-deploy-with-tensorflow/train-hyperparameter-tune-deploy-with-tensorflow.ipynb @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
 /sdk/python/resources/compute/compute.ipynb @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
 /sdk/python/resources/workspace/workspace.ipynb @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
-/sdk/python/resources/workspace/workspace.ipynb @joshharrin @seanyao1 @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1
```
Comment (sdgilley) [inline .github/CODEOWNERS:284]: again, leave this line as it was.  We don't want to delete when names are added to ours.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -183,12 +180,9 @@
 /cli/jobs/spark/serverless-spark-standalone-user-identity.yaml @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
 /cli/jobs/spark/spark-job-component.yml @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
 /cli/misc.sh @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
-/cli/monitoring/azureml-e2e-model-monitoring @sdgilley @msakande @Blackmist @nemanjarajic @ahughes-msft @yunjie-hub
```
Comment (sdgilley) [inline .github/CODEOWNERS:186]: leave these lines as they were.  We don't want to delete it when names are added to ours as was done here.

**Review by sdgilley — CHANGES_REQUESTED** [review]

**Pull request closed without merging**